main.js

```diff
@@ -36,6 +36,48 @@ var formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+function ODate(rootEl) {
```

Comment:
What made me change the casing was OBT, it throw errors during linting if the constructor function is not capitalised.